applications/bci_visualization/operators/reconstruction/normalize_operator.py

```
@@ -0,0 +1,163 @@
+        self._hard_coded_row_normalizers_cache: NDArray[np.float32] | None = None
+        self._hard_coded_normalized_jacobian_cache: NDArray[np.float32] | None = None
+
+    def setup(self, spec: OperatorSpec) -> None:
+        spec.input("batch")
+        spec.output("normalized")
+
+    def compute(
+        self,
+        op_input: InputContext,
+        op_output: OutputContext,
+        context: ExecutionContext,
```

Ruff (0.14.10)

Unused method argument: `context`

(ARG002)